packages/frameworks-solid-start/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.m.json",
```

Comment:
Curious, is `tsconfig.m.json` generated? If not, can we just put it's content directly in this one? The less config the better.

Reply:
Ye i really don't know why did I separate them, sure can do that